docker/openproblems-r-base/Dockerfile

```diff
@@ -14,14 +30,22 @@ RUN apt-get clean -y && apt-get autoremove -y
 # Install R packages
 RUN R -e "install.packages('renv'); renv::consent(TRUE)"
 COPY ./docker/openproblems-r-base/r_requirements.txt ./r_requirements.txt
-
-# Install r_requirements.txt
 RUN R -e "renv::install(scan('r_requirements.txt', sep='\n', what=character()))"
 
-# Install R kernel
-RUN echo 'IRkernel::installspec()' > /tmp/temp.R && Rscript /tmp/temp.R
+# Install kernelspec
+RUN R -e "IRkernel::installspec(user = FALSE);"
+
+# Install Python packages
+COPY ./docker/openproblems-python-extras/requirements.txt ./requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
are you installing python extras in the R base image here on purpose? This would increase the size of the base image quite dramatically and every image built on top of it... I would consider a new docker image for this, or at least use `r-extras`?